lib/rules/function-no-unknown/__tests__/index.mjs

```diff
@@ -27,6 +27,9 @@ testRule({
 		{
 			code: 'a { height: calc(10px*(5*(10 - 5))); }',
 		},
+		{
+			code: 'a { height: calc-size(auto); }',
```

Comment:
[suggestion] Using an example on the [CSS spec](https://drafts.csswg.org/css-values-5/#interp-calc-size), can you update the test code? E.g.

```suggestion
			code: 'a { height: calc-size(any, 0px); }',
```

Reply:
Fixed

Reply:
This can be replaced by a test case for `function-calc-no-unspaced-operator`.